test/linter/test-consistency.js

```diff
@@ -267,8 +281,10 @@ class ConsistencyChecker {
     );
   }
   /**
-   * @param {CompatStatement} compatData
-   * @return {string[]}
+   * Get all of the browsers with a version number in a feature
```

Comment:
Repeating myself, but I'd love if all of these ended with a full-stop.

Sometimes it happens that we start writing a comment, but get interrupted in doing so, and never get to finish it. The full-stop let's us detect those cases in review.

```suggestion
   * Get all of the browsers with a version number in a feature.
```